k8s/upgrades/0.4.0-0.5.0/oebs_update.sh

```diff
@@ -0,0 +1,36 @@
+#!/bin/bash
+
+#### GET PV NAME AS ARGUMENT #### 
+pv=$1
+
+#### GENERATE DEPLOY, REPLICASET AND CONTAINER NAMES FROM PV ####
```

Comment:
Samples/Examples of what happens before & after execution will be good to have in the comments.